bin/cli.js

```diff
@@ -58,7 +58,6 @@ function invoke(env) {
   env.modulePath = env.modulePath || env.knexpath || process.env.KNEX_PATH;
 
   const filetypes = ['js', 'coffee', 'ts', 'eg', 'ls'];
-  let pending = null;
```

Comment:
it wasn't used for anything?

Reply:
Yeah `pending` was assigned but was never used